scripts/ci/lib.sh

```diff
@@ -1409,6 +1414,26 @@ EOT
     hub-comment -type build -template-file "$tmpfile"
 }
 
+get_junit_parse_cli() {
+    # TODO !! Need to remove this commit hash before merging
+    go install github.com/stackrox/junit-parse@blugo/4e2c784
```

Comment:
Installing in CI can often lead to flakiness. It would be best to install this in the CI image to avoid the `go install` (stackrox/rox-ci-image:stackrox-test). However getting that image updated in CI is a major pain so I would rely on a future update to get the binary present. In the meantime you can `command -v` to check if it is present before installing. Something like: https://github.com/stackrox/stackrox/blob/d0cc2801cdd5373f6256d77aebe69f8cd83ae0c9/tests/upgrade/run.sh#L105